apps/api-extractor/src/collector/Collector.ts

```diff
@@ -495,8 +495,11 @@ export class Collector {
       if (entity.exportNames.has(idealNameForEmit)) {
```

Comment:
checked again and seems only exported entity are handled. if the entity with name "default" is not exported, it could also lead to invalid output.

e.g. 
```ts
// foo.d.ts
export { default as default } from 'bar';

// index.d.ts
import Foo from './foo';
export declare class A {
    prop: Foo; // <-- reference the entity with name "default", which is not exported
}
```

In this case the "default" will still be in the output... 😕

Reply:
Probably we should also handle it in the following code for unexported entity